### PR TITLE
adds hair and face y offsets variables for species

### PR DESCRIPTION
**Pull request by Militaires**
allows for species with different heights e.g to use the same hair and face system 

very naice very good.

tested and it works nice

### CODE REVIEW

**Comment by Supermichael777** [comment]
👌 MMMM THATS SOME GOOD SHIT

**Comment by PKPenguin321** [comment]
Giraffe-man species with variable neck length you can set in the lobby when

**Comment by ExcessiveUseOfCobblestone** [comment]
every round you survive as giraffe man your neck grows by 1 pt. If you die though it resets.

**Comment by WJohn** [comment]
do we have a system like this for inhands, too? it'd be really useful to not be forced to draw mob hands at the same spot in order to hold items (monkeys and xenos have their hands aligned with where inhands are, for example, but changing this would mean they wouldn't HAVE to)

**Comment by Militaires** [comment]
@wjohn no we dont that's what im doing next

**Comment by RemieRichards** [comment]
this affects all directions.
that's fine for Y which won't (realistically) be affected by Dirs, but it'll royally screw up X.

**Comment by Militaires** [comment]
@remierichards you can just add an override with switch(dir) and add the values you want and then call the rest of handle_hair

**Comment by RemieRichards** [comment]
no you can't, as handle_hair isn't called every time someone's dir changes (and shouldn't be)

**Comment by Militaires** [comment]
hmm @remierichard do you think i should add vars for east/west

**Comment by RemieRichards** [comment]
it still can't be handled appropriately, I would say ditch the X offset. (Y, as I said previously, should work fine)
Any races/mobs that DO require an X offset are probably so fugged up that I doubt the standard hair/eyes etc. would work anyway.

**Comment by Militaires** [comment]
fuck it ill just do that hold of the on

**Comment by Militaires** [comment]
ah fuk it

File: code/modules/mob/living/carbon/human/species.dm [hunk 1]
```diff
@@ -259,12 +264,15 @@
...
 			lips.color = H.lip_color
+			lips.pixel_y += face_y_offset
 			standing	+= lips
 
 		// eyes
 		if((EYECOLOR in species_traits) && HD && has_eyes)
 			var/image/img_eyes = image("icon" = 'icons/mob/human_face.dmi', "icon_state" = "eyes", "layer" = -BODY_LAYER)
 			img_eyes.color = "#" + H.eye_color
+			img_eyes.pixel_y += face_y_offset
+			img_eyes.pixel_x += face_x_offset
```
Comment (lzimann) [inline code/modules/mob/living/carbon/human/species.dm]: forgot one

**Comment by KorPhaeron** [comment]
Doesn't even compile

**Comment by PKPenguin321** [comment]
he missed 1 var kor, its a 1 line fix

**Comment by KorPhaeron** [comment]
then he can fix it easy and I'll reopen

**Comment by Militaires** [comment]
@KorPhaeron >doesnt compile >close

**Pull request merged**